nbgrader/coursedir.py

```diff
@@ -214,6 +214,29 @@ def _root_default(self):
         help=dedent(
             """
             List of file names or file globs to be ignored when copying directories.
+            Directories with matching names will be ignored too.
+            """
+        )
+    ).tag(config=True)
+
+    include = List(
+        [
+            "*",
+        ],
+        help=dedent(
+            """
+            List of file names or file globs: non matching files
+            will be ignored with a warning.
```

Comment:
Maybe say you need to use the DEBUG log level to see the warning?

Reply:
Ah, good point, forgot to update that part.

Reply:
Done, and polished a bit the option documentation at this occasion.

Reply:
Thanks!